provider/aws/formation/rack.json

```
@@ -2670,7 +2810,13 @@
               { "Name": "RACK", "Value": { "Ref": "AWS::StackName" } },
               { "Name": "STACK_ID", "Value": { "Ref": "AWS::StackId" } }
             ],
-            "Image": { "Fn::Sub": "convox/rack:${Version}" },
+            "Image":                 {
+              "Fn::If": [
+                "InstanceARM",
+                { "Fn::Sub": "convox:rack:${Version}-arm64" },
```

Typo on the image definition:
```suggestion
                { "Fn::Sub": "convox/rack:${Version}-arm64" },
```